server.R

```diff
@@ -288,6 +288,15 @@ server <- function(input, output) {
   
   # Filter data for within areas graph - variable name = within_areas_data_tab_1
   
+ within_areas_data_tab_1 <- reactive({
+   x <- add_pop_index(gender_selection= selected_gender_tab_1(), age_selection= input$age_choice_tab_1) %>%
+    filter(Council.Name == input$la_choice_tab_1() && 
```

Comment:
```suggestion
    filter(Council.Name == input$la_choice_tab_1 && 
```

Reply:
You don't need to brackets to call this because you're calling a reactive input using input$, you only need to brackets if you are calling something which has been stored as a reactive value